lib/prefs.cpp

```diff
@@ -234,7 +234,7 @@ void GLOBAL_PREFS::defaults() {
 #endif
     ram_max_used_busy_frac = 0.5;
 #ifdef ANDROID
-    ram_max_used_idle_frac = 0.5;
+    ram_max_used_idle_frac = 0.9;
```

Comment:
I think the code would be simpler if you just simply delete the android section.

Reply:
@adamradocz 
Yes, I agree. But I was waiting to hear the opinion of @AenBleidd about changing the value itself.
If he agree, I will change the code to be more simpler. 

@adamradocz Thank you for your time to review my PRs.